pal/samples/esp8266/sample_init.h

```diff
@@ -5,7 +5,13 @@
 #define SAMPLE_INIT_H
 #if defined(ARDUINO_ARCH_ESP8266)
   #define sample_init esp8266_sample_init
+  #define is_esp_board
   void esp8266_sample_init(const char* ssid, const char* password);
 #endif
+#if defined(ARDUINO_ARCH_ESP32)
+  #define sample_init esp32_sample_init
+  #define is_esp_board
+  void esp32_sample_init(const char* ssid, const char* password);
+#endif
```

Comment:
nit: can we end the `#endif`'s with the comment detailing how they started?
so
```c
#endif /* ARDUINO_ARCH_ESP32 */
```

Reply:
sure thing.